### PR TITLE
Quick Fix for issue on install CentOS error w/o rootPassword

### PR DESCRIPTION
**Pull request by cgx027**
Issue link: https://github.com/RackHD/RackHD/issues/36
Replace the 'null' default rootPassword.

The long term solution discuss is still undergoing at github issue
page. This is rather a short term fix to make it workable immediatly

### CODE REVIEW

File: lib/task-data/tasks/install-centos.js [hunk 1]
```diff
@@ -12,12 +12,12 @@ module.exports = {
         profile: 'install-centos.ipxe',
         hostname: 'localhost',
         comport: 'ttyS0',
-        domain: 'rackhd.github.com',
+        domain: 'rackhd',
```
Comment (heckj) [inline lib/task-data/tasks/install-centos.js]: Was reducing this to just RackHD intentional?

**Comment by heckj** [comment]
Question about one of the changes, but :+1: for the quick fix solution while we sort out longer term defaults

File: lib/task-data/tasks/install-centos.js (hunk 1 above)
Comment (cgx027) [inline lib/task-data/tasks/install-centos.js]: This was suggested by @yyscamper. This is also intended to keep consistent with the install-ESXi job where domain is set to 'rackhd'. It is suggested that having github.com in the domain makes rackhd less visible. I am open to this change. Let me konw if you are preferring to have github.com in the domain.

**Comment by cgx027** [comment]
Also copy @RackHD/corecommitters  and folks in Shanghai, @WangWinson , @yyscamper , @iceiilin , @anhou .

File: lib/task-data/tasks/install-centos.js (hunk 1 above)
Comment (heckj) [inline lib/task-data/tasks/install-centos.js]: It's fine, just wanted to make sure there was intention behind it. A single word as a domain is uncommon, but as a placeholder/reference example, it's perfectly acceptable

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**